io-engine/src/bdev/lvs.rs

```diff
@@ -0,0 +1,279 @@
+//! Allows creation of an lvs lvol through a URI specification rather than gRPC
+//! or direct function call. This is not intended for the usual product
+//! operation but for testing and benchmarking.
+//!
+//! # Uri
+//! lvol:///$name?size=$size&lvs=$lvs
```

Comment:
How this uri is used in io-engine-client CLI ? 
`./io-engine-client pool create lvol:///$name?size=$size&lvs=$lvs`

Reply:
Sorry.. understood.. It is part of PR description :)